swap_meet/clothing.py

```diff
@@ -1,2 +1,10 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, condition=0):
+        super().__init__(category = "Clothing", condition=condition)
```

Comment:
Stay consistent with your syntax.

```suggestion
        super().__init__(category="Clothing", condition=condition)
```